tutorials/cloudsdk-js-vdm-delete/cloudsdk-js-vdm-delete.md

```diff
@@ -141,10 +141,40 @@ function deleteBusinessPartnerAddress(businessPartnerId: string, addressId: stri
     });
 }
 ```
+
+As of version `1.7.0` of the SAP Cloud SDK, you can alternatively pass an entity to the `delete` method instead of the entity's key fields, like this:
```

Comment:
Give some guidance on when to use this approach instead ("if you already have an entity, for example, from a previous request"). We should not burden the reader with having to decide between the two versions without any hints.